example/memcached-operator/handler.go.tmpl

```diff
@@ -8,18 +8,29 @@ import (
 	v1alpha1 "github.com/example-inc/memcached-operator/pkg/apis/cache/v1alpha1"
 
 	"github.com/operator-framework/operator-sdk/pkg/sdk"
+	"github.com/prometheus/client_golang/prometheus"
 	appsv1 "k8s.io/api/apps/v1"
 	"k8s.io/api/core/v1"
 	apierrors "k8s.io/apimachinery/pkg/api/errors"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 	"k8s.io/apimachinery/pkg/labels"
 )
 
-func NewHandler() sdk.Handler {
-	return &Handler{}
+func NewHandler(m *Metrics) sdk.Handler {
+	return &Handler{
+    metrics: m,
```

Comment:
Can you run a `gofmt` on this file to fix the formatting? I think your editor was using spaces instead of tabs, which causes formatting issues with go. Many editors automatically fix go files, but this is `.tmpl` so editor miss that.